projects/js-packages/charts/src/providers/chart-context/private/get-chart-color.ts

```diff
@@ -48,10 +48,10 @@ const SATURATION_VARIATION_STEPS = 3;
 
 /**
  * Saturation increment per variation step
- * 15% increments provide noticeable but not jarring differences
- * Results in saturation levels: 60%, 75%, 90%
+ * 10% increments provide subtle variation while keeping colors muted
+ * Results in saturation levels: 45%, 55%, 65%
  */
-const SATURATION_INCREMENT = 15;
+const SATURATION_INCREMENT = 10;
```

Comment:
The previous saturation settings were resulting in every 3rd color being highly saturated, which overwhelmed the rest of the palette.